cmd/awscollector/main.go

```diff
@@ -17,13 +17,13 @@ package main // import "aws-observability.io/collector/cmd/awscollector"
 
 import (
 	"fmt"
+	"go.opentelemetry.io/collector/otelcol"
```

Comment:
nit: I think the ordering of imports needs to be adjusted. Does `go fmt` fix this?

Reply:
Ran `make fmt`.